components/automate-ui/src/app/page-components/user-table/user-table.component.html

```diff
@@ -13,23 +13,23 @@
     </app-authorized>
   </chef-toolbar>
   <ng-container *ngIf="showTable">
-    <chef-table>
-      <chef-thead>
-        <chef-tr>
-          <chef-th class="name-column">Name</chef-th>
-          <chef-th class="username-column">Username</chef-th>
-          <chef-th class="garbage-column"></chef-th>
-        </chef-tr>
-      </chef-thead>
-      <chef-tbody>
-        <chef-tr *ngFor="let user of users">
-          <chef-td class="name-column">
+    <chef-table-new>
+      <chef-table-header>
+        <chef-table-row>
+          <chef-table-header-cell class="name-column">Name</chef-table-header-cell>
+          <chef-table-header-cell class="username-column">Username</chef-table-header-cell>
+          <chef-table-header-cell class="garbage-column"></chef-table-header-cell>
+        </chef-table-row>
+      </chef-table-header>
+      <chef-table-body>
+        <chef-table-row *ngFor="let user of users">
+          <chef-table-cell class="name-column">
             <a [routerLink]="['/settings', 'users', user.id]">{{ user.name }}</a>
-          </chef-td>
-          <chef-td class="username-column">
+          </chef-table-cell>
+          <chef-table-cell class="username-column">
             {{ user.id }}
-          </chef-td>
-          <chef-td class="garbage-column">
+          </chef-table-cell>
+          <chef-table-cell class="garbage-column">
```

Comment:
Would like to suggest changing the CSS class to a more consistent name. 

Policy-details, also in this PR, uses "three-dot-entry". 
Policy-list uses "button-column".
Team-management uses "button-column".
Tokens-list uses "controls".
Projects-list uses "controls".

Maybe standardize on one of those?

Reply:
yes please. also is it needed? compliance tables with the control menu don't have special styling for that column

Reply:
We could make it a default of the table CSS but maybe that's cleaner to do in the three-dot menu itself after we pull it back into angular?

Reply:
Sounds like we are gonna look at a general approach to this CSS fix once we get the three-dot menu in, but I will standardize the class names for now.